Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM jrei/systemd-debian:10
+FROM ghcr.io/raspap/raspap-docker
```

Comment:
I think we can apply the line 3 curl on top of the bullseye/bookworm work done here: https://github.com/RaspAP/raspap-docker/pull/19, so the raspap content will be inside the tag `ghcr.io/raspap/raspap-docker`

Reply:
I think this line should be unchanged once https://github.com/RaspAP/raspap-docker/pull/22/files is merged.

Reply:
yes, otherwise this will create an infinite loop, where this docker image will build upon itself

Reply:
fixed by new commit